motionplan/solverFrame.go

```diff
@@ -148,21 +149,22 @@ func newSolverFrame(fs frame.FrameSystem, solveFrameName, goalFrameName string,
 	}
 
 	var ptgs []tpspace.PTGSolver
-	anyPTG := false     // Whether PTG frames have been observed
-	anyNonzero := false // Whether non-PTG frames
+	anyPTG := false // Whether PTG frames have been observed
+	// TODO(nf): uncomment
+	// anyNonzero := false // Whether non-PTG frames
 	for _, movingFrame := range frames {
 		if ptgFrame, isPTGframe := movingFrame.(tpspace.PTGProvider); isPTGframe {
 			if anyPTG {
 				return nil, errors.New("only one PTG frame can be planned for at a time")
 			}
 			anyPTG = true
 			ptgs = ptgFrame.PTGSolvers()
-		} else if len(movingFrame.DoF()) > 0 {
-			anyNonzero = true
-		}
-		if anyNonzero && anyPTG {
```

Comment:
are ok with this being removed?

Reply:
No, this still needs to be here

Reply:
moved into planManager planRelativeWaypoint

Reply:
CheckPlan will have a ptgFrame + another frame with DoF >0, moving this allows us to circumvent this issue.
specifically this is important when we are working with the wrapperFrame which has a ptgFrame and poseFrame